tests/eibc_fulfillment_test.go

```diff
@@ -1789,6 +1789,281 @@ func TestEIBCFulfillment_ThirdParty_Wasm(t *testing.T) {
+		}).
+		AddLink(test.InterchainLink{
+			Chain1:  dymension,
+			Chain2:  rollapp2,
+			Relayer: r2,
+			Path:    anotherIbcPath,
+		})
+
+	rep := testreporter.NewNopReporter()
+	eRep := rep.RelayerExecReporter(t)
+
+	err = ic.Build(ctx, eRep, test.InterchainBuildOptions{
+		TestName:         t.Name(),
+		Client:           client,
+		NetworkID:        network,
+		SkipPathCreation: true,
+
+		// This can be used to write to the block database which will index all block data e.g. txs, msgs, events, etc.
+		// BlockDatabaseFile: test.DefaultBlockDatabaseFilepath(),
+	})
+	require.NoError(t, err)
+
+	CreateChannel(ctx, t, r1, eRep, dymension.CosmosChain, rollapp1.CosmosChain, ibcPath)
+	CreateChannel(ctx, t, r2, eRep, dymension.CosmosChain, rollapp2.CosmosChain, anotherIbcPath)
+
+	walletAmount := math.NewInt(1_000_000_000_000)
+
+	// Create some user accounts on both chains
+	users := test.GetAndFundTestUsers(t, ctx, t.Name(), walletAmount, dymension, dymension, rollapp1, rollapp2)
+
+	// Wait a few blocks for relayer to start and for user accounts to be created
+	err = testutil.WaitForBlocks(ctx, 5, dymension, rollapp1, rollapp2)
+	require.NoError(t, err)
+
+	// Get our Bech32 encoded user addresses
+	dymensionUser, marketMaker, rollappUser, rollapp2User := users[0], users[1], users[2], users[3]
+
+	dymensionUserAddr := dymensionUser.FormattedAddress()
+	marketMakerAddr := marketMaker.FormattedAddress()
+	rollappUserAddr := rollappUser.FormattedAddress()
+	rollapp2UserAddr := rollapp2User.FormattedAddress()
+
+	// Assert the accounts were funded
+	testutil.AssertBalance(t, ctx, dymension, dymensionUserAddr, dymension.Config().Denom, walletAmount)
+	testutil.AssertBalance(t, ctx, dymension, marketMakerAddr, dymension.Config().Denom, walletAmount)
+	testutil.AssertBalance(t, ctx, rollapp1, rollappUserAddr, rollapp1.Config().Denom, walletAmount)
+	testutil.AssertBalance(t, ctx, rollapp2, rollapp2UserAddr, rollapp2.Config().Denom, walletAmount)
+
+	transferAmount := math.NewInt(1_000_000)
+	multiplier := math.NewInt(10)
+
+	eibcFee := transferAmount.Quo(multiplier) // transferAmount * 0.1
+
+	dymChannel_ra1, err := r1.GetChannels(ctx, eRep, rollapp1.Config().ChainID)
+	require.NoError(t, err)
+	require.Equal(t, 1, len(dymChannel_ra1))
+	channDymRollApp1 := dymChannel_ra1[0]
+	require.NotEmpty(t, channDymRollApp1.ChannelID)
+
+	dymChannel_ra2, err := r2.GetChannels(ctx, eRep, rollapp2.Config().ChainID)
+	require.NoError(t, err)
+	require.Equal(t, 1, len(dymChannel_ra2))
+	channDymRollApp2 := dymChannel_ra2[0].Counterparty
+	require.NotEmpty(t, channDymRollApp2.ChannelID)
+
+	rollapp1_params := rollappParam{
+		rollappID: rollapp1.Config().ChainID,
+		channelID: channDymRollApp1.ChannelID,
+		userKey:   dymensionUser.KeyName(),
+	}
+
+	rollapp2_params := rollappParam{
+		rollappID: rollapp2.Config().ChainID,
+		channelID: channDymRollApp2.ChannelID,
+		userKey:   dymensionUser.KeyName(),
+	}
+
+	triggerHubGenesisEvent(t, dymension, rollapp1_params)
```

Comment:
we can add rollapp2_params2 like `triggerHubGenesisEvent(t, dymension, rollapp1_params, rollapp1_params)`so don't need to add more `triggerHubGenesisEvent` below